platform/feature-api/communication/src/remote-pod/remote.ts

```diff
@@ -105,64 +105,7 @@ type PodBackend = ObjectBackendSpec<{
     endpoint(): EndpointBackend;
 }>;
 
-class FileStats implements Stats {
-    static of(stats: Stats): FileStats {
-        if (
-            stats.getSize !== undefined &&
-            stats.getName !== undefined &&
-            stats.getTime !== undefined &&
-            stats.getId !== undefined
-        ) {
-            return new FileStats(
-                stats.isFile(),
-                stats.isDirectory(),
-                stats.getTime(),
-                stats.getSize(),
-                stats.getName(),
-                stats.getId()
-            );
-        } else {
-            return new FileStats(
-                stats.isFile(),
-                stats.isDirectory(),
-                "",
-                0,
-                "",
-                ""
-            );
-        }
-    }
-
-    constructor(
-        readonly file: boolean,
-        readonly directory: boolean,
-        readonly time: string,
-        readonly size: number,
-        readonly name: string,
-        readonly id: string
-    ) {}
-    isFile(): boolean {
-        return this.file;
-    }
-    isDirectory(): boolean {
-        return this.directory;
-    }
-    getTime(): string {
-        return this.time;
-    }
-    getSize(): number {
-        return this.size;
-    }
-    getName(): string {
-        return this.name;
-    }
-    getId(): string {
-        return this.id;
-    }
-}
-
 export const podBubblewrapClasses: Classes = {
-    "@polypoly-eu/remote-pod.FileStats": FileStats,
```

Comment:
Pretty sure this is the culprit. Having a look.

Reply:
Yeah, so seems like pod.js is trying to deserialise the value it gets from iOS into this object, but it doesn't exist since it was eliminated. For other value objects we return (e.g. `ExternalFile`), we just do a map, I'm looking into whether we can just do the same here.

Reply:
This one fixes it on iOS: https://github.com/polypoly-eu/polyPod/pull/1140

Got some troubles with my Android build, but I'll get that working and see if it's also an issue there. In my mind, it should be.

Reply:
Turns out Android was already ignoring the `FileStats` class, so this is an iOS only issue. My PR should resolve it.